guizlet/frontend-interview
# [RISK-25] Updating deps + addressing security vulnerabilities for all scaffolds

Addressing security vulnerabilities for all scaffolds by updating deps and adding resolutions when necessary. Also added `yarn.lock` for Angular 8 repo.

Steps taken:
- `yarn install` for Vanilla JS, Angular 1/8, React scaffolds
- For security vulnerabilities, run `npm audit fix` to address any issues. Two manual fixes needed addressing were manually resolving dependencies from the Parcel bundler to be at a version that resolves the security vulnerability:
-- [minimist](https://github.com/parcel-bundler/parcel/issues/4978)
-- [node-forge](https://github.com/parcel-bundler/parcel/issues/5145#issuecomment-706215764)

---

**Test plan:**
- Run `npm install`/`yarn install`, navigate to `localhost:1234` and see that Vanilla JS scaffold works
- Run `npm install`/`yarn install`, navigate to `localhost:1234` and see that React scaffold works
- Run `npm install`/`yarn install`, navigate to `localhost:1234` and see that Angular 1 scaffold works
- Run `npm install`/`yarn install`, navigate to `localhost:4200` and see that Angular 8 scaffold works
